archlinuxcn/lib32-llvm-git/PKGBUILD

```diff
@@ -12,7 +12,7 @@ url="https://llvm.org/"
 license=('custom:Apache 2.0 with LLVM Exception')
 makedepends=('git' 'cmake' 'ninja' 'python' 'python2'
              'lib32-gcc-libs' 'lib32-libffi' 'lib32-libunwind'
-             'lib32-libxml2' 'lib32-zlib')
+             'lib32-libxml2' 'lib32-zlib' 'gcc-multilib' 'llvm-git')
```

Comment:
No `llvm-git` is not a build dependency. And gcc-multilib should not be included.